lib/active_remote/serializers/protobuf.rb

```diff
@@ -1,24 +1,26 @@
+require "active_remote/typecasting"
+
 module ActiveRemote
   module Serializers
     module Protobuf
       extend ActiveSupport::Concern
 
       TYPECASTER_MAP = {
```

Comment:
This is the data adapter's typecasting.  The typecasters that were causing performance issues are the user typecasters.  Since I was ripping out the active_attr typecasting system I needed to change this file, since out data adapter typecasters were relying on the active_attr typecasters.

As a side note, most of this is probably not necessary at all since the protobuf library is already handling most of the typecasting.